pmm-app/tests/verifyQAN_test.js

```diff
@@ -3,7 +3,8 @@ Feature('QAN Dashboard');
 Before(async (I, qanPage, adminPage) => {
   I.Authorize();
 
-  I.amOnPage(qanPage.url + "?from=now-5m&to=now");
+  I.amOnPage(`${qanPage.url}?from=now-5m&to=now`);
+  //this two elements should be changed after new QAN
```

Comment:
```suggestion
  // TODO: these two elements should be changed once new QAN is merged
```
Pls make this a `TODO` item

Reply:
Also pls leave a note as to how we want to change them